problem-set/MoveBeepers.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * File: MoveBeepers.java
+ * ----------------------------
+ * MoveBeepers
+ * ცარიელ სამყაროში კარელი დგას 1x1 უჯრაზე. 5x1 უჯრაზე დევს 10 ცალი ბრილიანტი.
+ * კარელმა ათივე ბრილიანტი უნდა გადაიტანოს 6x1 უჯრაზე.
+ * დეტალური ახსნა შეგიძლიათ იხილოთ MoveBeepers.md ფაილში.
+ */
+
+import stanford.karel.*;
+
+public class KarelProgramSample extends Karel {
+	private void moveFourTimes() {
+		// Karel stands on the cell 1x1.
+		for (int i = 1; i <= 4; i++) {
```

Comment:
ეს არ იციან ამ დროს, ჯობია ყველა ციკლია ასე გამოვიყენოთ:
```suggestion
		for (int i = 0; i < 4; i++) {
```